packages/page-alert/src/page-alert.tsx

```diff
@@ -0,0 +1,69 @@
+import { PropsWithChildren } from 'react';
+import { Box, Flex } from '@ag.ds-next/box';
+import { globalPalette } from '@ag.ds-next/core';
+import {
+	AlertFilledIcon,
+	InfoFilledIcon,
+	SuccessFilledIcon,
+	WarningFilledIcon,
+} from '@ag.ds-next/icon';
+
+import { PageAlertTitle } from './page-alert-title';
+
+export type PageAlertTone = 'success' | 'error' | 'warning' | 'info';
+
+export type PageAlertProps = PropsWithChildren<{
+	title?: string;
+	tone: PageAlertTone;
+}>;
+
+export const PageAlert = ({
+	children,
+	title,
+	tone,
+	...props
+}: PageAlertProps) => {
+	const { fg, bg } = pageAlertToneMap[tone];
+	const Icon = icons[tone];
+
+	return (
+		<Flex
+			rounded
+			css={{
+				backgroundColor: bg,
+			}}
+			{...props}
+		>
+			<Flex
+				padding={0.75}
+				alignItems="center"
+				palette="dark"
+				css={{
+					borderTopLeftRadius: 4,
```

Comment:
```suggestion
					borderTopLeftRadius: tokens.borderRadius,
```

Reply:
Sorry about that, didn't put a todo on it